src/production/phases/BundlePhase.ts

```diff
@@ -30,6 +36,8 @@ export async function BundlePhase(productionContext: IProductionContext) {
       await attachWebIndex(ctx, bundles);
     }
 
+    ctx.log.stopStreaming();

```

Comment:
not needed...

https://github.com/fuse-box/fuse-box/blob/grand-refactor/src/core/helpers/finalizeFusebox.ts#L6